test/corpus/expressions.txt

```diff
@@ -503,33 +513,36 @@ WHERE
           name: (identifier)))
       (where
         (keyword_where)
-        (binary_expression
+        expression: (binary_expression
           left: (binary_expression
             left: (binary_expression
               left: (binary_expression
                 left: (field
                   name: (identifier))
-                operator: (keyword_like)
+                operator: (operator)
                 right: (literal))
               operator: (keyword_and)
               right: (binary_expression
                 left: (field
                   name: (identifier))
-                operator: (keyword_not)
-                operator: (keyword_like)
+                operator: (operator
+                  (keyword_not)
+                  (keyword_like))
```

Comment:
this is another remaining weirdness: the `like` is resolved to an `(operator)` while `not like` is `(operator (keyword_not) (keyword_like))` (and you can see the `similar to` operations below behaving....similarly).

Replacing this line:
```
[$._not_like, 'pattern_matching'],
```
with
```
[seq($.keyword_not, $.keyword_like), 'pattern_matching'],
```
results in `not like` also resolving to an opaque `(operator)`. Should we try to prefer surfacing the keywords?

Reply:
I posit: yes, and it turned out to be pretty manageable at a second look -- even brought the total state count down by a few dozen in spite of introducing a few more node types!